MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+struct AlertPresenter {
+    private var model: AlertModel
+    weak var viewController: UIViewController?
+    
+    init(modelToShowAlert model: AlertModel) {
+        self.model = model
+    }
+    
+    func showAlert() {
+        let alertController = UIAlertController.init(title: model.title, message: model.message, preferredStyle: .alert)
+        
+        alertController.addAction(UIAlertAction(title: model.buttonText, style: .cancel, handler: {_ in self.model.completion()}))
```

Comment:
давай комплишены разбивать как минимум пробелом после фигурной скобки, как максимум – начинать с новой строки:
```suggestion
        alertController.addAction(UIAlertAction(title: model.buttonText, style: .cancel, handler: { _ in 
            self.model.completion()
        }})
```